runtime/compiler/trj9/x/env/J9CPU.cpp

```diff
@@ -31,64 +31,52 @@ namespace J9
 namespace X86
 {
 
-const char *
-CPU::getX86ProcessorVendorId(TR::Compilation *comp)
-   {
-   return comp->fej9()->getX86ProcessorVendorId();
-   }
-
-uint32_t
-CPU::getX86ProcessorSignature(TR::Compilation *comp)
-   {
-   return comp->fej9()->getX86ProcessorSignature();
-   }
-
-uint32_t
-CPU::getX86ProcessorFeatureFlags(TR::Compilation *comp)
-   {
-   return comp->fej9()->getX86ProcessorFeatureFlags();
-   }
-
-uint32_t
-CPU::getX86ProcessorFeatureFlags2(TR::Compilation *comp)
+bool
+CPU::testOSForSSESupport()
```

Comment:
Why not remove this function completely?

Reply:
For this function:
there is a OMR::X86::CPU::testOSForSSESupport(), which returns false.
and a J9::X86::CPU::testOSForSSESupport(), which returns true.
There are two calls of this function in x/codegen/OMRCodeGenerator.cpp. If we remove the J9 version, they will return false because OMR version will get called. Do you want the J9 version to be removed? and change the OMR version to return true maybe? I'm not understanding why we want this removed.

Reply:
I see, I guess we have to wait until OMR sorts out what to do with ```testOSForSSESupport()``` first. Let's keep it as is for now.